tests/src/testadaptivestepsizing.cpp

```diff
@@ -104,21 +85,24 @@ auto KLShellAndAdaptiveStepSizing(const PathFollowingType& pft, const std::vecto
 
   BoundaryPatch<decltype(gridView)> neumannBoundary(gridView, neumannVertices);
 
-  auto neumannBoundaryLoad = [&](auto& globalCoord, auto& lamb) {
+  auto nBL = [&](auto& globalCoord, auto& lamb) {
```

Comment:
if we shorten the name then may be comments are needed to understand the meaning of `nBL`? Even though it is used only in tests, but for our future reference may be comments are then more descriptive?